railties/lib/rails/generators/generated_attribute.rb

```diff
@@ -239,6 +239,30 @@ def options_for_migration
           end
         end
       end
+
+      def attr_options_to_s
```

Comment:
Does this method needs to be public?
```suggestion
      def print_attribute_options
```

Reply:
renamed the method and made it private